pom.xml

```diff
@@ -27,15 +27,15 @@
     </scm>
 
     <properties>
-        <bamboo.version>6.8.1</bamboo.version>
+        <bamboo.version>6.9.1</bamboo.version>
```

Comment:
Let's change the version here to the latest Bamboo release version.

Reply:
Done